t/plugin/basic-auth.t

```diff
@@ -20,6 +20,7 @@ repeat_each(2);
 no_long_string();
 no_root_location();
 no_shuffle();
+log_level('info');
```

Comment:
@mangoGoForward please take a check whether this change is needed. Thanks.

Reply:
Seems unnecessary, removed.